### PR TITLE
docs: tutorials->tutorial and rename flask page

### PR DESCRIPTION
**Pull request by cjdcordeiro**
- [x] Have you signed the [CLA](http://www.ubuntu.com/legal/contributors/)?

-----

 - rename Tutorials to Tutorial
  >  Tutorial because there should really be just one tutorial. One learning experience, even if in multiple sections.

 - rename the Flask tutorial .rst file in order to achieve a smaller URL
 - the docs are also versioned and without language, so this PR also removes the `/en/` from the links in the codebase

### CODE REVIEW

**Comment by tigarmo** [comment]
Did you already change the readthedocs versioning scheme? Because now eg https://documentation.ubuntu.com/rockcraft/en/stable/reference/rockcraft.yaml is a 404 and that breaks links printed by stable Rockcraft.

**Comment by cjdcordeiro** [comment]
@tigarmo thanks for reverting it.

File: .github/ISSUE_TEMPLATE/docs.yaml [hunk 1]
```diff
@@ -32,7 +32,7 @@ body:
     attributes:
       label: Documentation location
       description: Location of the corresponding documentation page (e.g., file, section, URL), if applicable
-      placeholder: ex. https://canonical-rockcraft.readthedocs-hosted.com/en/latest/tutorials/hello-world/
+      placeholder: ex. https://canonical-rockcraft.readthedocs-hosted.com/latest/tutorial/hello-world/
```
Comment (sergiusens) [inline .github/ISSUE_TEMPLATE/docs.yaml:35]: why don't you use the friendly ubuntu link here too?

Comment (cjdcordeiro) [inline .github/ISSUE_TEMPLATE/docs.yaml:35]:
```suggestion
      placeholder: ex. https://documentation.ubuntu.com/rockcraft/stable/tutorial/hello-world/
```

Comment (cjdcordeiro) [reply .github/ISSUE_TEMPLATE/docs.yaml]: yep

**Review by tigarmo — CHANGES_REQUESTED** [review]
I'm fine with the changes in principle, but unless this PR also add redirects it'll break the links on older versions of Rockcraft

**Comment by cjdcordeiro** [comment]
Which older versions have tutorial links in the codebase? This current version doesn't have any

**Review by tigarmo — APPROVED** [review]
oh you're right! Thanks

**Review by mr-cal — APPROVED** [review]

**Pull request merged**